ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -166,7 +165,7 @@ static int getSchemaVersion(InstanceConfig instanceConfig) {
    */
   static List<String> getSealedReplicas(InstanceConfig instanceConfig) {
     List<String> sealedReplicas = instanceConfig.getRecord().getListField(ClusterMapUtils.SEALED_STR);
-    return sealedReplicas == null ? Collections.emptyList() : sealedReplicas;
+    return sealedReplicas == null ? new ArrayList<>() : sealedReplicas;
```

Comment:
Just curious why.

Reply:
Collections.emptyList() is immutable, we cannot call list.remove() method.